AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.html

```diff
@@ -76,38 +76,42 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
 
       <p class="govuk-body govuk-!-margin-bottom-5"></p>
 
-      <div [ngClass]="{ 'govuk-form-group--error': judgeEmailInvalid || !isValidEmail }" *ngIf="displayEmailField">
-        <legend id="judgeEmail-error" *ngIf="judgeEmailInvalid || !isValidEmail">
+      <div [ngClass]="{ 'govuk-form-group--error': judgePhoneInvalid }">
+        <legend id="judgePhone-error" *ngIf="judgePhoneInvalid">
```

Comment:
Moved email below phone as it is the only field that appears for one and not the other